WebhookNotification/handler.ts

```diff
@@ -251,12 +259,22 @@ export const getWebhookNotificationHandler = (
           const webhookNotification =
             errorOrWebhookNotification.right.channels.WEBHOOK;
 
+          const userProfile = await pipe(
+            userProfileReader({
+              fiscalCode: message.fiscalCode
+            }),
+            TE.getOrElse(err => {
+              throw new Error(err.title);
```

Comment:
it's better to throw the err title or the err message?

Reply:
Not sure, since this error is needed for replaying the operation, I thought it was not useful to write detailed info.